app/views/conference_registrations/_questions.html.haml

```diff
@@ -1,8 +1,8 @@
-- @conference.questions.each do |q|
-  = f.inputs 'Additional Info' do
+= f.inputs 'Additional Info' do
+  - @conference.questions.each do |q|
     - if q.question_type.id == 1  || q.question_type.id == 2 # yes/no or single choice
       = f.input :qanswers, :collection => q.qanswers, :as => :select, :input_html => { :multiple => false, class: 'col-sm-10' }, label: q.title, :include_blank => "Please make your choice",
                 :member_label => Proc.new {|a| a.answer.title}
     - if q.question_type.id == 3  # multiple choice
-      = f.input :qanswers, :collection => q.qanswers, :as => :check_boxes, :input_html => { class: 'col-sm-2' }, :label => false,
+      = f.input :qanswers, :collection => q.qanswers, :as => :check_boxes, :input_html => { class: 'col-sm-2' }, :label => q.title,
                 :member_label => Proc.new {|a| a.answer.title}
```

Comment:
Yes I am on master branch. Idk which extra file rubocop is checking when travis runs it, as locally it checkes 359 files
![screenshot from 2015-03-20 21 02 41](https://cloud.githubusercontent.com/assets/7680662/6754746/97a4cf4a-cf44-11e4-8e94-44efaafeef74.png)

So this time, I changes two files:
`views/users/show.html.haml` -> `views/users/show.html.rb`
`views/users/edit.html.haml` -> `views/users/edit.html.rb`

Files checked increased to 361. I don't think rubocop is suppose to check haml files either, cause it would generate unwanted error like ` unexpected token error` for `%h`. haml-lint is suppose to do that, their documentation does that we can integrate it with rubocop but I am not sure how exactly we do it.